.github/workflows/main.yml

```diff
@@ -34,7 +34,10 @@ jobs:
         run: yarn build
 
       - name: Sign plugin
-        run: npx @grafana/toolkit plugin:sign --rootUrls ${{ secrets.PLUGIN_ROOT_URLS }}
+        # sign plugin for public use
+        run: npx @grafana/toolkit plugin:sign
+        # sign plugin for private use
+        # run: npx @grafana/toolkit plugin:sign --rootUrls ${{ secrets.PLUGIN_ROOT_URLS }}
```

Comment:
nope, see https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/#sign-a-public-plugin

Reply:
I'm not sure if we should maintain separate releases, one for public and one for private use :thinking: If the plugin will be added to the catalogue, there's no need for us to "manually" install the plugin anymore and the root URLs won't be necessary anyways ^^

Reply:
not sure, let's see once we get it there, maybe the private signature is no longer required

Reply:
AFAIK if the plugin is in the grafana catalogue, it can be installed from "official sources", meaning the private signature is no longer necessary. the same goes for the helm chart: instead of downloading, unpacking and installing it, the plugin can just be referenced as dependency (or something, not sure how that'll work) and installed as part of the boot process.